docs/specs/clients/sign/client-api.md

```diff
@@ -74,6 +74,24 @@ abstract class Client {
     reason: Reason;
   }): Promise<void>;
 
+  // ---------- Wallet Authentication Methods ----------------------------------------------- //
+
+  // for proposer to request wallet authentication
+  public abstract authenticate(params: {
```

Comment:
will it work?

Reply:
it use to be RequestParams
```
    public let domain: String
    public let chainId: String
    public let nonce: String
    public let aud: String
    public let nbf: String?
    public let exp: String?
    public let statement: String?
    public let requestId: String?
    public let resources: [String]?
```

Reply:
probably we can simplify 🤔

Reply:
we can, but saw that I missed some required parameters so for the specs I'm listing now all available